lms/djangoapps/teams/views.py

```diff
@@ -1461,10 +1460,10 @@ def post(self, request):
                     'add_method': 'joined_from_team_view' if user == request.user else 'added_by_another_user'
                 }
             )
-        except AlreadyOnTeamInCourse:
+        except AlreadyOnTeamInTeamset:
             return Response(
                 build_api_error(
-                    ugettext_noop(u"The user {username} is already a member of a team in this course."),
+                    ugettext_noop(u"The user {username} is already a member of a team in this teamset."),
```

Comment:
'u' prefix is no longer needed in Python 3+ as all strings are unicode.

Reply:
Updated.